### PR TITLE
Create a response for the Product Item object

### PR DESCRIPTION
**Pull request by wvega**
# Summary

This PR implements the `API\Catalog\Product_Item\Response` class.

## QA

- [x] `vendor codecept run integration integration tests/integration/API/Catalog/Product_Item/ResponseTest.php` pass

### CODE REVIEW

File: includes/API/Catalog/Product_Item/Response.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	/**
+	 * Gets the Product Item group ID.
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @return string
+	 */
+	public function get_group_id() {
+
+		$product_group = '';
```
Comment (deivamagalhaes) [inline includes/API/Catalog/Product_Item/Response.php:34]:
```suggestion
		$product_group_id = '';
```

File: includes/API/Catalog/Product_Item/Response.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+		if ( isset( $this->response_data->product_group->id ) ) {
+			$product_group = $this->response_data->product_group->id;
```
Comment (deivamagalhaes) [inline includes/API/Catalog/Product_Item/Response.php:37]:
```suggestion
			$product_group_id = $this->response_data->product_group->id;
```

File: includes/API/Catalog/Product_Item/Response.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		}
+
+		return $product_group;
```
Comment (deivamagalhaes) [inline includes/API/Catalog/Product_Item/Response.php:40]:
```suggestion
		return $product_group_id;
```

File: tests/integration/API/Catalog/Product_Item/ResponseTest.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Tests\API\Catalog\Product_Item;
+
+use SkyVerge\WooCommerce\Facebook\API\Catalog\Product_Item\Response;
+
+/**
+ * Tests the Connection class.
```
Comment (deivamagalhaes) [inline tests/integration/API/Catalog/Product_Item/ResponseTest.php:8]: Needs to be updated.

Comment (wvega) [reply tests/integration/API/Catalog/Product_Item/ResponseTest.php]: I finally enabled spell checking in Visual Studio Code for the typos 😅, now I need something for this kind of copy🍝

Thank you for catching that one! Fixed in cef61cd.

File: includes/API/Catalog/Product_Item/Response.php (hunk 1 above)
Comment (wvega) [reply includes/API/Catalog/Product_Item/Response.php]: Good idea. Thank you for the suggestions!

**Pull request merged**